extensions/ql-vscode/src/view/common/ActionButton/ActionButton.module.css

```
@@ -0,0 +1,61 @@
+/* Styles have been copied from https://vscode-elements.github.io/elements-lite/components/action-button/configurator/ */
+.actionButton {
+  align-items: center;
+  background-color: transparent;
+  border-color: transparent;
+  border-style: solid;
+  border-width: 1px;
+  border-radius: 5px;
+  color: var(--vscode-foreground);
+  display: inline-flex;
+  cursor: pointer;
+  padding: 0;
+  user-select: none;
+}
+
+.actionButton:disabled {
+  color: var(--vscode-disabledForeground);
+  cursor: default;
+  pointer-events: none;
+}
+
+.actionButton .codicon,
```

Does this actually apply to Codicons? By default, this generates a class name since we're inside of a CSS module, so it would only apply to Codicons that specifically include this class. I think the intention is for this to apply to all Codicons:

```suggestion
.actionButton :global(.codicon),
```

However, if it currently works and doesn't cause any styling issues, I think you can also just remove this.